w3-hw1-vy/cypress/pages/bookstore/bookstorePage.js

```diff
@@ -0,0 +1,69 @@
+export const bookstore = {
+
+    // BOOK STORE PAGE
+
+    TXT_BOOK_NAME_COL: ".rt-table .rt-tr-group .rt-td:nth-child(2) a",
+    TXT_SEARCH_BOX: "#searchBox",
+    TXT_CURRENT_PAGE: ".-pageJump input",
+    TXT_TOTAL_PAGES: ".-totalPages",
+    BTN_PAGE: ".-pagination button",
+    BTN_PROFILE: "#item-3 span",
+
+    clickProfile(){
+        cy.get('span').contains('Profile').click({force:true});
```

Comment:
You knew the trick here but you didn't apply for `TXT_BOOK_NAME_COL` so you are using index in your locator